src/docs/pages/component-docs/components/score-card/examples/with-actions.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+
+export const WITH_ACTIONS = `<blui-score-card style="width: 350px" class="with-actions" headerTitle="Header Actions">
+   <ng-container blui-action-items>
+        <mat-icon>search</mat-icon>
+        <mat-icon>mail</mat-icon>
+        <mat-icon>notifications</mat-icon>
```

Comment:
You typically see these icons here instead:

```suggestion
        <mat-icon>star</mat-icon>
        <mat-icon>settings</mat-icon>
        <mat-icon>more_vert</mat-icon>
```